modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SuggestionResourceTest.java

```diff
@@ -6,14 +6,123 @@
 package com.liferay.portal.search.rest.resource.v1_0.test;
 
 import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.journal.model.JournalArticle;
+import com.liferay.journal.test.util.JournalTestUtil;
+import com.liferay.layout.test.util.LayoutTestUtil;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.model.Layout;
+import com.liferay.portal.kernel.test.util.GroupTestUtil;
+import com.liferay.portal.search.rest.client.dto.v1_0.Suggestion;
+import com.liferay.portal.search.rest.client.dto.v1_0.SuggestionsContributorConfiguration;
+import com.liferay.portal.search.rest.client.dto.v1_0.SuggestionsContributorResults;
+import com.liferay.portal.search.rest.client.pagination.Page;
 
-import org.junit.Ignore;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
 import org.junit.runner.RunWith;
 
 /**
  * @author Petteri Karttunen
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class SuggestionResourceTest extends BaseSuggestionResourceTestCase {
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
```

Comment:
The testGroup is available from the base class so no need to create it here. The base class also cleans it up.